codemp/ui/ui_main.c

```diff
@@ -7440,7 +7437,7 @@ static void UI_BuildServerDisplayList(int force) {
 	lanSource = UI_SourceForLAN();
 
 	if (force) {
-		numinvisible = 0;
+		// numinvisible = 0;
```

Comment:
isn't checking in commented-out code a bad practice, generally?

Reply:
Yeah, but removing some of these lines would have left empty if blocks followed by non-empty else-if blocks. In that case I'd rather there be some context as to why the control flow is how it is, which an empty block does not convey

Reply:
Removed now.